modules/transactions/src/main/java/org/apache/ignite/internal/tx/TxManager.java

```diff
@@ -129,21 +130,19 @@ CompletableFuture<Void> finish(
     );
 
     /**
-     * Sends cleanup request to the specified primary replica.
+     * Sends cleanup request to the primary nodes of each one of {@code partitions}.
```

Comment:
Could you please reformulate it? I'd use something like: Sends cleanup request to the cluster nodes that hosts primary replicas for the enlisted partitions.